polaris-terraform/ui-terraform/app-service.tf

```diff
@@ -105,13 +106,14 @@ resource "azurerm_linux_web_app" "as_web_polaris" {
       app_settings["REACT_APP_FEATURE_FLAG_REDACTION_LOG"],
       app_settings["REACT_APP_FEATURE_FLAG_REDACTION_LOG_UNDER_OVER"],
       app_settings["REACT_APP_FEATURE_FLAG_FULL_SCREEN"],
+      app_settings["REACT_APP_FEATURE_FLAG_NOTES"],
       app_settings["REACT_APP_LOCAL_STORAGE_EXPIRY_DAYS"],
       app_settings["REACT_APP_GATEWAY_BASE_URL"],
       app_settings["REACT_APP_GATEWAY_SCOPE"],
       app_settings["REACT_APP_IS_REDACTION_SERVICE_OFFLINE"],
       app_settings["REACT_APP_PRIVATE_BETA_SIGN_UP_URL"],
       app_settings["REACT_APP_PRIVATE_BETA_USER_GROUP"],
-      app_settings["REACT_APP_REDACTION_LOG_USER_GROUP"],
```

Comment:
@aceltuk i have remove the unused variable , and added two new variables , hope that is ok.

Reply:
That should be totally fine